gems/pending/Gemfile

```diff
@@ -23,7 +23,9 @@ gem "httpclient",              "~>2.7.1",           :require => false
 gem "image-inspector-client",  "~>1.0.3",           :require => false
 gem "iniparse",                                     :require => false
 gem "kubeclient",              "=1.2.0",            :require => false
-gem "linux_admin",             "~>0.17.0",          :require => false
+gem "linux_admin",             "~>0.18.0",          :require => false
+gem "sshkey",                  "~>1.8.0",           :require => false
```

Comment:
To clarify,  `gems/pending/spec/util/miq_ssh_util_v2_spec.rb` would fail if you have the sshkey in the main Gemfile because the gems/pending tests do NOT include that Gemfile.  They use their own gems/pending/Gemfile.  You would have to move the dependency of MiqSshUtilV2 to the gems/pending/Gemfile as you have done here.

If we choose to keep the code in `create_deployment` in the main rails app (see comments below), the dependency should live in the main Gemfile.